controllers/serviceaccount/interface.go

```diff
@@ -1,35 +1,75 @@
 package serviceaccount
 
 import (
+	"bytes"
 	"context"
+	cesappcore "github.com/cloudogu/cesapp-lib/core"
+	v2 "github.com/cloudogu/k8s-dogu-operator/v2/api/v2"
+	"github.com/cloudogu/k8s-dogu-operator/v2/controllers/exec"
 	"github.com/cloudogu/k8s-registry-lib/config"
+	corev1 "k8s.io/api/core/v1"
 )
 
-type SensitiveDoguConfigProvider interface {
-	GetSensitiveDoguConfig(ctx context.Context, doguName string) (SensitiveDoguConfig, error)
+//nolint:unused
+//goland:noinspection GoUnusedType
+type sensitiveDoguConfigProvider interface {
+	GetSensitiveDoguConfig(ctx context.Context, doguName string) (sensitiveDoguConfig, error)
 }
 
-type SensitiveDoguConfigSetter interface {
+type sensitiveDoguConfigSetter interface {
 	Set(ctx context.Context, key, value string) error
 }
 
-type SensitiveDoguConfigGetter interface {
+type sensitiveDoguConfigGetter interface {
```

Comment:
🚫 **[checkstyle]** <unused> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>type `sensitiveDoguConfigGetter` is unused
<!-- __reviewdog__:ChA2Y2EzZjRlZGI2ZTg0ZGNiEgpjaGVja3N0eWxl -->